src/rollup/types.d.ts

```diff
@@ -1,5 +1,6 @@
 import * as ESTree from 'estree';
 import { EventEmitter } from 'events';
+import Graph from '../Graph';
```

Comment:
The idea is that all types that are exposed to the user are strictly controlled by us and originate from types.d.ts. We simply cannot expose the Graph because it exposes everything: Internal AST data structures, internal module data structures etc. Isn't there a way to create a well-defined JSON data structure that contains the information you desire?
If we expose Graph, then from then on, virtually ALL Rollup releases will be breaking changes. We should really try to find a better way.